src/components/containers/Sidebar/Sidebar.js

```diff
@@ -1,12 +1,31 @@
 import React, { Component } from 'react';
 
 import './Sidebar.css';
+import github from '../../../images/github.png';
+import info from '../../../images/blueinfo.png';
+import youtube from '../../../images/youtube.png';
+import bluedocumentation from '../../../images/bluedocumentation.png';
+import purple from '../../../images/Purple.png';
+
 
 class Sidebar extends Component {
   render() {
     return (
       <div className='sidebar'>
-        <p>This is a sidebar</p>
+        <div className="IconSizes">
+        <a href="https://github.com/awesome-api/stt-tooling">
+          <button><img src = {github}/>GitHub</button>
```

Comment:
If we have an `<a>` already, do we need a `<button>` inside of that?

Reply:
It definitely looks cleaner, when wrapped in a button element. And it keeps the impression that it's still a menu with boxes